tools/verdict/com.ge.research.osate.verdict/plugin.xml

```diff
@@ -99,7 +117,21 @@
 	            style="push"
 	            tooltip="Click to change CRV Settings">
 	       </command>
-		</menu>                                   	                         
+		</menu>      
+		<menu id="GsnMenu" label="&amp;Assurance Cases (GSN)" >
```

Comment:
Can you remove the shorthand "(GSN)" in the top level menu to be consistent?

Reply:
Done